src/classes/HiFiMixerSession.ts

```diff
@@ -271,6 +276,7 @@ export class HiFiMixerSession {
         this._lastSuccessfulInputAudioMutedValue = false;
         this.onMuteChanged = onMuteChanged;
         this._getUserFacingConnectionState = getUserFacingConnectionState;
+        this._disableReconnect = false;
```

Comment:
`_disableReconnect` should also be set to false when the client intentionally connects, i.e. through `HiFiCommunicator.connectToHiFiAudioAPIServer`

Reply:
fixed